### PR TITLE
Fix for GET  /services & added new API path - /services/search

### PR DESCRIPTION
**Pull request by dmpl1**
New endpoint for search and provided exact match for the services list.

### Requirements
* Defect: GET  /services retrieves multiple services during search since it uses CONTAINS instead of exact match in DynamoDB query

### Description of the Change
* Created a new endpoint for search. 
* Provided exact match for services list.

### CODE REVIEW

File: core/jazz_services/swagger/swagger.json [hunk 1]
```diff
@@ -542,6 +542,537 @@
...
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            }
+          }
+        }
+      },
+      "post": {
```
Comment (devsatishm) [inline core/jazz_services/swagger/swagger.json]: Why do we need `post`?

**Comment by devsatishm** [comment]
Looks good overall. Please fix the Travis build error (unit tests).

File: core/jazz_services/swagger/swagger.json (hunk 1 above)
Comment (dmpl1) [reply core/jazz_services/swagger/swagger.json]: Post is not needed. Changes updated.

File: core/jazz_services/components/crud/getSearchList.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
+// =========================================================================
```
Comment (raghits) [inline core/jazz_services/components/crud/getSearchList.js]: getSearchList.js is practically the same as getList.js except probably for one line - reuse the code and make it DRY?

File: core/jazz_services/swagger/swagger.json [hunk 3]
```diff
@@ -542,6 +542,332 @@
         }
       }
     },
+    "/services/search": {
+      "options": {
+        "description": "CORS Support",
```
Comment (raghits) [inline core/jazz_services/swagger/swagger.json]: Had to enable CORS manually in API Gateway for /services/search.

File: core/jazz_services/swagger/swagger.json [hunk 4]
```diff
@@ -542,6 +542,332 @@
...
+        "consumes": [
```
Comment (raghits) [inline core/jazz_services/swagger/swagger.json]: See POST method for /services/search which is not needed. 
image

Comment (dmpl1) [reply core/jazz_services/swagger/swagger.json]: Removed POST

File: core/jazz_services/components/crud/getSearchList.js (hunk 2 above)
Comment (dmpl1) [reply core/jazz_services/components/crud/getSearchList.js]: Addressed with conditions.

**Pull request merged**